lib/Controller/MessagesController.php

```diff
@@ -575,28 +592,39 @@ public function setFlags(int $id, array $flags): JSONResponse {
 	 *
 	 * @param int $accountId
 	 * @param string $folderId
-	 * @param int $id
+	 * @param array $ids
 	 *
 	 * @return JSONResponse
 	 *
 	 * @throws ClientException
 	 * @throws ServiceException
 	 */
-	public function destroy(int $id): JSONResponse {
+	public function deleteMessages(array $ids): JSONResponse {
+		$uids = [];
+		$message = $this->mailManager->getMessage($this->currentUserId, $ids[0]);
+		$firstMailbox = $this->mailManager->getMailbox($this->currentUserId, $message->getMailboxId());
+		$firstAccount = $this->accountService->find($this->currentUserId, $firstMailbox->getAccountId());
```

Comment:
You try to get the account without catching a possible ClientException.

See the following excerpt from `AccountService.find()`:
```php
    /**
	 * @param string $uid
	 * @param int $id
	 *
	 * @return Account
	 * @throws ClientException
	 */
	public function find(string $uid, int $id): Account {
```